scootapi/client/smoke_test_cmd.go

```diff
@@ -61,6 +61,41 @@ func (c *Client) runSmokeTest(cmd *cobra.Command, args []string) error {
 		}()
 	}
 
+	// current status of every job
+	jobStatusMap := make(map[string]scoot.Status)
+	// get updated statuses as they're sent
+	go func() {
+		for {
+			jobAndStatus := <-ch
+			for job, status := range jobAndStatus {
+				jobStatusMap[job] = status
+			}
+		}
+	}()
+
+	ticker := time.NewTicker(time.Millisecond * 250)
+	// job id's grouped by status
+	statusJobMap := make(map[scoot.Status][]string)
+Loop:
+	for range ticker.C {
+		for job, status := range jobStatusMap {
+			// populate statusJobMap
+			statusJobMap[status] = append(statusJobMap[status], job)
+		}
+		for status, jobs := range statusJobMap {
+			sort.Sort(sort.StringSlice(jobs))
+			fmt.Println(status, ":", jobs)
```

Comment:
Should a timed out job have a status of Status_ROLLED_BACK? Or should it have an explicit timed out status?